packages/components/src/components/crypto-money/crypto-money.jsx

```diff
@@ -0,0 +1,34 @@
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import Popover from '../popover';
```

Comment:
Please avoid using `index.js` of the components inside components package. This will lead to duplicate css wherever we use this component. You can use the component directly, like: `Import Popover from '../popover/popover.jsx`